build.sh

```diff
@@ -200,13 +201,13 @@ while test "$1" != ""; do
 done
 
 # Suck in local vars
-. ${BUILDER_TOOLS}/builder_defaults.sh
+. "${BUILDER_TOOLS}"/builder_defaults.sh
```

Comment:
I agree, but it does not harm either. Do you prefer `"${BUILDER_TOOLS}/builder_defaults.sh"` instead?